Cargo.toml

```diff
@@ -14,7 +14,7 @@ members = [
 
 # Dependencies we want to keep the same between workspace members
 [workspace.dependencies]  
-wsts = "7.0"
+wsts = { version = "7.0", default-features = false }
```

Comment:
This is from the wsts Cargo.toml:

```toml
[features]
default = ["with_p256k1_bindgen"]
with_p256k1_bindgen = ["p256k1/with_bindgen"]
```

So is that sub-feature not necessary for the signer's needs? Or is `bindgen` in p256k1 just not needed?

Reply:
bindgen is one of the major problems here and needs to be disabled

Reply:
No, it needs to be enabled for platforms that `libclang-sys` supports static linking on.  That includes Linux with glibc, but not Linux with musl.